src/api/spec/controllers/webui/watched_items_controller_spec.rb

```diff
@@ -64,8 +64,9 @@
 
         subject { put :toggle_watched_item, params: { package_name: package, project_name: package.project } }
 
-        it 'raises an exception' do
-          expect { subject }.to raise_error(NotFoundError)
+        it 'redirects and assigns flash error' do
+          expect(subject).to redirect_to(root_url)
+          expect(subject.request.flash[:error]).to eq('This page is not accesible unless you have the "New watchlist" flag enabled in the beta program')
```

Comment:
To match my previous comment:

```suggestion
          expect(subject.request.flash[:error]).to eq('This page is not accessible unless you enabled the "New watchlist" beta feature in the beta program.')
```